docs/source/index.rst

```diff
@@ -98,7 +103,8 @@ Tests
     
     ```roslaunch mslquad vel_ctrl_test.launch```
```

Comment:
these are all outdated and we're going to be doing a proper testing thing next so i don't think we need to add this here.

Reply:
@k2shah Since our current codebase does use, or can use these, I think it's good to keep them for now. As I want to make sure the current functionality is relayed to the user, even if it's not what we'll be using in the future. But if they actually don't work, then we should remove them.

Reply:
I've never used them or seen them been used so i'd not be willing to have other people try them